src/vm/functions/conversions.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2013-2020 Blockstack PBC, a public benefit corporation
+// Copyright (C) 2020 Stacks Open Internet Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+use vm::costs::cost_functions::ClarityCostFunction;
+use vm::costs::runtime_cost;
+use vm::errors::{check_argument_count, CheckErrors, InterpreterResult as Result};
+use vm::representations::SymbolicExpression;
+use vm::types::BufferLength;
+use vm::types::SequenceSubtype::{BufferType, StringType};
+use vm::types::StringSubtype::ASCII;
+use vm::types::TypeSignature::SequenceType;
+use vm::types::{SequenceData, TypeSignature, Value};
+use vm::{apply, eval, lookup_function, Environment, LocalContext};
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub enum EndianDirection {
```

Comment:
Do we need a specific `EndianDirection` type?  Could we get away with instead defining four different `conversion_fn` functions that simply wrapped `::from_le_bytes` and `::from_be_bytes` for `u128` and `i128`?

Reply:
The endian direction has a use apart from this, because we need to know which side to do the padding on.